video_player version to ">=0.4.0 <=0.4.1"

ponnamkarthik opened the pull request.

brianegan commented:
Looks great, thanks for contributing! Could we please widen the version constraint?

Should be safe to use ">=0.4.0 <=0.5.0"

PonnamKarthik commented:
https://pub.dartlang.org/packages/video_player#-changelog-tab-


yes it is safe

Changelog of video_player

0.4.1
    Updated Flutter SDK constraint to reflect the changes in v0.4.0.

0.4.0
    Breaking change. Removed the VideoPlayerController constructor
    Added two new factory constructors VideoPlayerController.asset and VideoPlayerController.network to respectively play a video from the Flutter assets and from a network uri.

brianegan commented:
Awesome, thanks so much :)
